packages/@aws-cdk/aws-stepfunctions-tasks/test/sagemaker/create-transform-job.test.ts

```diff
@@ -249,3 +249,62 @@ test('pass param to transform job', () => {
     },
   });
 });
+
+test('create transform job with instance type as a parameter', () => {
```

Comment:
```suggestion
test('create transform job with instance type supplied as JsonPath', () => {
```

nit: small tweak to make it clearer what we are testing here

Reply:
Thanks for reviewing! Fixed